CONTRIBUTING.md

```diff
@@ -51,6 +51,15 @@ python -m black .
 
 [black]: <https://black.readthedocs.io/en/stable/>
 
+## GitHub Codespace
+
+A pre-configured development environment is available as a GitHub Codespaces dev container.
+
+After starting the container, make sure to activate the virtual environment:
+```
+source .venv/bin/activate
```

Comment:
out of curiosity, would this be needed if we didn't have `deactivate` in the post create command? (I'm not very familiar with codespaces)

Reply:
yes, post create runs in a different terminal -- the standard way of doing this [is complicated](https://docs.github.com/en/codespaces/setting-your-user-preferences/personalizing-github-codespaces-for-your-account#dotfiles), but lemme try something, might find a way around